library/src/main/kotlin/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -75,6 +81,43 @@ public object Chucker {
         NotificationHelper(context).dismissNotifications()
     }
 
+    /**
+     * Export the chucker transactions to a file.
+     *
+     * This must be called on a background thread.
+     *
+     * @param context Application context
+     * @param maxTransactions Maximum number of transactions to return in the file. Passing null
+     * means no limit on the number of transactions
+     * @param startTimestamp The timestamp to read transactions from. Passing null means
+     * transactions will not be limited by timestamp
+     * @return The content uri of a file with the transactions in
+     */
+    @JvmStatic
+    public fun writeTransactions(
+        context: Context,
+        maxTransactions: Long?,
+        startTimestamp: Long?,
+    ): Uri? {
+        RepositoryProvider.initialize(context)
+        val transactions =
+            RepositoryProvider.transaction().getTransactions(maxTransactions, startTimestamp)
+        if (transactions.isEmpty()) {
+            Toast
+                .makeText(context, R.string.chucker_export_empty_text, Toast.LENGTH_SHORT)
+                .show()
+            return null
+        }
```

Comment:
This method should not show UI operations. It's up to the consumer of this method to do so if they wish (say if the URI returned is null).